src/app/services/views/ServiceListView.vue

```diff
@@ -1,176 +1,146 @@
 <template>
-  <RouteView>
-    <RouteTitle
-      :title="t('services.routes.items.title')"
-    />
-    <AppView>
-      <ContentWrapper>
-        <template #content>
-          <DataOverview
-            :selected-entity-name="service?.name"
-            :page-size="PAGE_SIZE_DEFAULT"
-            :error="error"
-            :is-loading="isLoading"
-            :empty-state="EMPTY_STATE"
-            :table-data="tableData"
-            :table-data-is-empty="tableData.data.length === 0"
-            :next="nextUrl"
-            :page-offset="pageOffset"
-            @table-action="loadEntity"
-            @load-data="loadData"
-          />
+  <RouteView
+    v-slot="{ route }"
+    name="services-list-view"
+  >
+    <DataSource
+      v-slot="{data, error}: ServiceInsightCollectionSource"
+      :src="`/${props.mesh}/services?page=${props.page}&size=${props.size}`"
+    >
+      <AppView>
+        <template #title>
+          <h2>
+            <RouteTitle
+              :title="t('services.routes.items.title')"
+              :render="true"
+            />
+          </h2>
         </template>
-
-        <template #sidebar>
-          <ServiceSummary
-            v-if="service !== null"
-            :service="service"
-            :external-service="externalService"
-          />
-        </template>
-      </ContentWrapper>
-    </AppView>
+        <KCard>
+          <template #body>
+            <AppCollection
+              data-testid="service-collection"
+              :empty-state-title="t('common.emptyState.title')"
+              :empty-state-message="t('common.emptyState.message', { type: 'Services' })"
+              :headers="[
+                { label: 'Name', key: 'name' },
+                { label: 'Type', key: 'serviceType' },
+                { label: 'Address', key: 'addressPort' },
+                { label: 'Status', key: 'status' },
+                { label: 'DP proxies (online / total)', key: 'online' },
+                { label: 'Actions', key: 'actions', hideLabel: true },
+              ]"
+              :page-number="props.page"
+              :page-size="props.size"
+              :total="data?.total"
+              :items="data?.items"
+              :error="error"
+              @change="route.update"
+            >
+              <template #name="{ row: item }">
```

Comment:
**Question**: Why all this renaming to `item`?

Reply:
Because its more generic than `row`, ie. if I want to copy/paste something from inside the slot elsewhere (i.e. to somewhere where it isn't a `row`) the word `item` still works. I don't have to copy/paste and then rename the variable from row to something else.

Reply:
Is this a **Question** or a blocker?